schemas/package.json

```diff
@@ -239,6 +242,42 @@
       "required": [
         "description"
       ]
+    },
+    "lexicalModels": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/lexicalModel"
+      }
+    },
+    "lexicalModel": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "version": {
+          "type": "string"
+        },
+        "rtl": {
```

Comment:
Should this be `isRTL` to be consistent with `#/definitions/ModelInfo`?

Reply:
Consistency would be a nice thing to have. Sadly I've chosen to use `rtl` to be consistent with `package.json#/definitions/keyboards/rtl`... so at least the fields are relatively consistent within a single file.